packages/perseus-editor/src/widgets/dropdown-editor.tsx

```diff
@@ -208,51 +210,54 @@ class DropdownEditor extends React.Component<Props> {
                                         )}
                                         value={i}
                                     />
-                                    <input
-                                        type="text"
+                                    <TextField
+                                        value={choice.content}
                                         ref={"editor" + i}
+                                        aria-label={`Choice ${i + 1} content`}
+                                        disabled={editingDisabled}
+                                        style={{
+                                            backgroundColor:
+                                                choiceBackgroundColor,
+                                        }}
```

Comment:
The wonder blocks TextField looks so much better!

Unfortunate nit: I think it causes the last field to have the invalid red background on load, rather than waiting for a change.

<img width="1086" height="604" alt="image" src="https://github.com/user-attachments/assets/788c0e56-db3a-4676-bf2a-149f6c55b00f" />